.github/workflows/cleardown-terraform-state.yaml

```diff
@@ -0,0 +1,44 @@
+name: Cleardown Terraform state
+
+on:
+  workflow_call:
+    inputs:
+      github_environment:
+        description: "Defines the Github environment in which to pull environment variables from"
+        required: true
+        type: string
+      deployment_workspace:
+        description: 'Name of the terraform workspace to spin up the service infrastructure'
```

Comment:
V trivial but in this instance the description is a bit misleading - tearing down not spinning up

Reply:
changed to 'tear down'